src/modals/NewFlow.tsx

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import ReactModal, { Styles } from "react-modal";
+import styled from "@emotion/styled";
+import { useDispatch, useSelector } from "react-redux";
+import { closeModal, showModal } from "../redux/global/global.actions";
+import { createNewFlow, duplicateFlow } from "../redux/flows/flows.actions";
+import { selectExampleFlowsKeyEntryPairs } from "../redux/flows/flows.selectors";
+
+const style: Styles = {
+  overlay: {
+    backgroundColor: "rgba(38, 50, 56, 0.5)",
+  },
+  content: {
+    backgroundColor: "#EBE8E8",
+    border: "none",
+    bottom: "auto",
+    maxHeight: "80%", // set height
+    left: "50%",
+    position: "fixed",
+    right: "auto",
+    top: "50%", // start from center
+    transform: "translate(-50%,-50%)", // adjust top "up" based on height
+    maxWidth: "44rem",
+    overflow: "hidden",
+  },
+};
+
+const CloseModal = styled.div`
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+  font-size: 2rem;
+  color: #ffffff;
+  float: right;
+  line-height: 100%;
+`;
+const CreateOptionsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 1rem 0 1rem;
+  margin-bottom: 1rem;
+`;
+
+const CreateOption = styled.div`
+  cursor: pointer;
+  background-color: #ffffff;
+  width: 12rem;
+  height: 12rem;
+  margin-top: 1rem;
+  margin-bottom: 0.5rem;
+  margin-right: 1rem;
```

Comment:
The top and bottom rows are not spaced evenly and the whole grid of buttons is offset more to the left because of the margins. I would suggest just using `margin: 0.5rem;` to get consistent margins.

Reply:
done